autoload/ale/python.vim

```diff
@@ -25,8 +25,7 @@ function! ale#python#FindProjectRootIni(buffer) abort
         \|| filereadable(l:path . '/tox.ini')
         \|| filereadable(l:path . '/mypy.ini')
         \|| filereadable(l:path . '/pycodestyle.cfg')
-        \|| filereadable(l:path . '/flake8.cfg')
-        \|| filereadable(l:path . '/.flake8rc')
```

Comment:
A number of projects and linter plugins search for `.flake8rc`, so I'd keep this too.

Reply:
I searched the history of the `flake8` project and it has never supported `.flake8rc`, from what I can tell.

I conducted a hasty [Google search: ".flake8rc"](https://www.google.com/search?q=".flake8rc"&oq=".flake8rc") and identified the following (potential) projects that may be using this are:

- https://github.com/AtomLinter/linter-flake8
- https://github.com/flycheck/flycheck
- https://github.com/w0rp/ale

Ale is using this in the documentation and within this function.

For `linter-flake8`, it looks like it is another documentation bug (https://github.com/AtomLinter/linter-flake8/blob/875c273d3affe253478b804a51441688a0b2d03e/package.json#L27), from search https://github.com/AtomLinter/linter-flake8/search?q=flake8rc&unscoped_q=flake8rc.

For `flycheck`, it appears they have a `flycheck-flake8rc`-specific configuration file documented in (https://github.com/flycheck/flycheck/blob/47174a12fd84a685f36019632838c73b4813b66d/doc/languages.rst).